.github/workflows/moodle-release.yml

```diff
@@ -0,0 +1,20 @@
+#
+# Whenever version.php is changed, add the latest version
+# to the Moodle Plugins directory at https://moodle.org/plugins
+#
+name: Releasing in the Plugins directory
+
+on:
+  push:
+    branches:
+      - MOODLE_310_STABLE
+    paths:
+      - 'version.php'
+
+jobs:
+  workflow_group_310_plus_release:
```

Comment:
Please rename this to just `release`

https://github.com/catalyst/catalyst-moodle-workflows#how-to-call-reusable-workflow-for-plugin-moodle-release